test/e2e/common/common.go

```diff
@@ -7,5 +7,6 @@ package common
 const (
 	RetinaPort int = 10093
 	// netObsRGtag is used to tag resources created by this test suite
-	NetObsRGtag = "-e2e-netobs-"
+	NetObsRGtag   = "-e2e-netobs-"
+	TestNamespace = "kube-system-test"
```

Comment:
Allowing namespace to be specified is useful, but should change the e2e away from kube-system?

For Azure case the Prometheus collector will scrape a service in kube-system, and debugging that in the future might be invisible for test authoring should we decide to add an e2e for that later

https://github.com/Azure/prometheus-collector/blob/20936204e18004eb162333504fa4a6c7f049d482/otelcollector/configmapparser/default-prom-configs/networkobservabilityRetinaDefaultDs.yml#L15

Reply:
Hmm I see now. I looked around and there seems to be other places where `kube-system` namespace is relied on, here are some examples:
* https://github.com/microsoft/retina/blob/main/get-certs.sh#L21 
* https://github.com/microsoft/retina/blob/main/deploy/legacy/prometheus/retina/create-cm.sh#L3 
For the scope of this PR, I'll revert the E2E testing in a different namespace. Extra refactoring and tests would be required to ensure that retina is not dependent on the `kube-system` namespace. Thanks for the heads up @matmerr